Blockzilla/AppDelegate.swift

```diff
@@ -40,7 +41,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         UserDefaults.standard.set(false, forKey: "WebKitLocalStorageEnabledPreferenceKey")
 
         // Set up our custom user agent.
-        UserAgent.setup()
+        userAgent = UserAgent(userDefaults: UserDefaults.standard)
```

Comment:
Do we still need this since we're using the singleton to instantiate `WebViewController`?

Reply:
good call this call also be removed.